spec/integration/htty/cli/commands/reuse_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'htty/cli/commands/reuse'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.